packages/server/src/database/service.ts

```diff
@@ -30,7 +24,7 @@ export class DatabaseService extends Service {
   }
 
   private loadPoolConfig() {
-    let config = this.configService.current.database?.pool
+    let config = undefined
```

Comment:
IMHO, you could remove this line and simply put line 37 inside the try-catch. It would be cleaner that way